src/WebJobs.Extensions.OpenAI.CosmosDBNoSqlSearch/CosmosDBNoSqlSearchConfigOptions.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
need samples for each of the language for this search provider

minimum is csharp ooproc (dotnet-isolated) with README.md

Additionally, it should be added to build files.